WordPress/Classes/ViewRelated/Blog/BlogDetailsViewController.m

```diff
@@ -1026,6 +1026,42 @@ - (NSString *)tableView:(UITableView *)tableView titleForHeaderInSection:(NSInte
     return detailSection.title;
 }
 
+- (UIView *)tableView:(UITableView *)tableView viewForHeaderInSection:(NSInteger)sectionNum {
+    BlogDetailsSection *section = [self.tableSections objectAtIndex:sectionNum];
+    if (section.showQuickStartMenu) {
+        return [self quickStartHeaderWithTitle:section.title];
+    } else {
+        return [super tableView:tableView viewForHeaderInSection:sectionNum];
+    }
+}
+
+- (UIView *)quickStartHeaderWithTitle:(NSString *)title
+{
+    NSString *removeTitle = NSLocalizedString(@"Remove Next Steps", @"Title for action that will remove the next steps/quick start menus.");
+    NSString *removeMessage = NSLocalizedString(@"Removing Next Steps will hide all tours on this site. This action cannot be undone.", @"Explanation of what will happen if the user confirms this alert.");
+    NSString *confirmationTitle = NSLocalizedString(@"Remove", @"Title for button that will confirm removing the next steps/quick start menus.");
+    NSString *cancelTitle = NSLocalizedString(@"Cancel", @"Cancel button");
+
+    UIAlertController *removeConfirmation = [UIAlertController alertControllerWithTitle:removeTitle message:removeMessage preferredStyle:UIAlertControllerStyleAlert];
+    [removeConfirmation addCancelActionWithTitle:cancelTitle handler:nil];
+    [removeConfirmation addDefaultActionWithTitle:confirmationTitle handler:^(UIAlertAction * _Nonnull action) {
+        [[QuickStartTourGuide find] removeFrom:self.blog];
+    }];
+
+    UIAlertController *removeSheet = [UIAlertController alertControllerWithTitle:nil message:nil preferredStyle:UIAlertControllerStyleActionSheet];
+    [removeSheet addDestructiveActionWithTitle:removeTitle handler:^(UIAlertAction * _Nonnull action) {
+        [self presentViewController:removeConfirmation animated:YES completion:nil];
+    }];
+    [removeSheet addCancelActionWithTitle:cancelTitle handler:nil];
+
+    BlogDetailsSectionHeaderView *view = [[[NSBundle mainBundle] loadNibNamed:[BlogDetailsSectionHeaderView classNameWithoutNamespaces] owner:nil options:nil] firstObject];
```

Comment:
Just a suggestion. The new header should be a `UITableViewHeaderFooterView` instead of a `UIView`. You can then register it with the `registerNib: forHeaderFooterViewReuseIdentifier:` and dequeue it with `dequeueReusableHeaderFooterViewWithIdentifier` avoiding to load it from the main bundle.
What do you think?

Reply:
I'll try that. tbh I just copied the pattern we've been using elsewhere, which I assume had a good reason (but I have no idea of 😜)